vscode/test/benchmark/datasets/typescript/infers-type-open-file/config.json

```
@@ -0,0 +1,8 @@
+{
+  "entryFile": "generate.ts",
+  "openFiles": ["generate.ts", "types.ts"],
+  "closedFiles": [],
+  "solutionFile": "solution.ts",
+  "testFile": "test.ts",
+  "testCommand": "ts-node"
```

@taras-yemets Just an FYI, I've updated the main branch to require the file is provided explicitly for this (as you may not always want a `testFile`, like @slimsag mentioned for Go)

```suggestion
  "testCommand": "ts-node test.ts"
```